package.json

```diff
@@ -24,26 +24,22 @@
   ],
   "scripts": {
     "build": "tsc",
+    "check": "tsc --noEmit",
+    "clean": "git clean -fXd",
     "test": "jest",
     "docs": "typedoc"
   },
-  "jest": {
-    "testEnvironment": "jsdom",
```

Comment:
Previously I needed to add this in order to make `fetch` work in the tests. Has something changed here?

Reply:
That's still needed. Moved it into a config file instead. (But forgot to push it - was listed in gitignore).